### PR TITLE
E501 - line too long (> 79 characters)

### PR DESCRIPTION
**Pull request by guedou**
Please don't merge ey!

The latest big #1277 PR. It used `autopep8` to fix lines lengths. When needed, I did some manual adjustements to keep comments.

### CODE REVIEW

File: scapy/arch/pcapdnet.py [hunk 1]
```diff
@@ -347,12 +366,14 @@ def send(self, x):
 
                 def __del__(self):
                     pcap.close(self.pcap)
-            open_pcap = lambda *args, **kargs: _PcapWrapper_libpcap(*args, **kargs)
+            open_pcap = lambda *args, **kargs: _PcapWrapper_libpcap(
+                *args, **kargs)
```
Comment (gpotter2) [inline scapy/arch/pcapdnet.py]: I don’t really like the way it get fixed. For instance, I find it way more understandable the way it was previously on this line...

File: scapy/arch/pcapdnet.py [hunk 2]
```diff
@@ -321,9 +338,11 @@ def __init__(self, device, snaplen, promisc, to_ms, monitor=False):
                         if pcap.pcap_set_rfmon(self.pcap, 1) != 0:
                             warning("Could not set monitor mode")
                         if pcap.pcap_activate(self.pcap) != 0:
-                            raise OSError("Could not activate the pcap handler")
+                            raise OSError(
+                                "Could not activate the pcap handler")
```
Comment (gpotter2) [inline scapy/arch/pcapdnet.py]: Same here for instance, it’s get weird

**Comment by guedou** [comment]
I spent quite some time on this PR, yet I am not super happy with the results. Another linter might do a better job.

**Comment by gpotter2** [comment]
Sorry if you spent time on that :/

File: scapy/arch/pcapdnet.py (hunk 1 above)
Comment (p-l-) [reply scapy/arch/pcapdnet.py]: In such cases, I'd rather have the closing parenthesis after another line break:
```Python
        open_pcap = lambda *args, **kargs: _PcapWrapper_libpcap(
            *args, **kargs
        )
```

Comment (guedou) [reply scapy/arch/pcapdnet.py]: `autopep8` does not do it this way. I could try to find another tool, but it might be a pain.

**Comment by guedou** [comment]
Another way to "fix" this error are:
- consider that 79 characters is not enough and extend the line length in `tox.ini`
- consider that some lines can be more than 79 characters long, let's say 90, fix everything below 90, and use noqa otherwise.

**Comment by b-x** [comment]
I suggest to set in tox.ini max-line-length = 100 or even 120.
The code should be readable and 79 characters in my opinion is too restrictive.

**Comment by guedou** [comment]
FTR.

flake8  currently reports 3106 E501 errors:
```
   1402 80
    608 90
    536 100
    270 110
    155 120
     43 130
     21 140
     19 150
     15 160
      9 270
      6 170
      6 180
      6 220
      3 290
      2 210
      1 190
      1 200
      1 230
      1 250
      1 280
```

**Comment by p-l-** [comment]
I think we should try to enforce 79 max chars on new code (let's say it's a rule in the project), without fixing it on existing code?

**Comment by guedou** [comment]
It would be a lot of work but that’s possible.

Could you have a look a the current PR and explain what you want to change and how ?

**Comment by p-l-** [comment]
What I mean is we do not change anything right now, and we try to enforce the 79 chars limit to new code.

**Comment by guedou** [comment]
Sounds good to me =)

**Comment by guedou** [comment]
@p-l- done! This PR is a mix of automatic and manual fixes.

**Comment by guedou** [comment]
Ready to be reviewed! A big thanks to unit tests =)

**Comment by gpotter2** [comment]
I’m Ok with that.

Let’s say that every time someone edits one of those lines, he must fix it in an elegant way

**Comment by p-l-** [comment]
@gpotter2 what @guedou did was exactly what I meant, so that's perfect :)

**Pull request merged**